creativecommons/vocabulary
# Changed the spelling mistake of Ancillary

## Fixes
- Fixes #184 by [Raja-Abrar-Khan](https://github.com/Raja-Abrar-Khan)

## Description
I corrected the spelling of "Ancillary" in all files where the term was misspelled, including the docs/index.html file and all other relevant files in the project folder. This update ensures consistent usage across the project and improves the professionalism of the website's UI.

## Technical details

.Updated the spelling of "Ancillary" in:
  -docs/index.html (lines 207 and 248)
  -src/css/vocabulary.css
  -All other files across the project where the incorrect spelling appeared, including HTML, CSS, and any other source files using 
   the term.
## Tests

To verify this PR:

1. Go through the website and check any sections where the term "Ancillary" is displayed to confirm the correction.
2. Review the project to ensure that all references to the CSS class with the incorrect spelling have been updated.

## Screenshots

![Screenshot 2024-10-13 225411](https://github.com/user-attachments/assets/e9682beb-4ce0-4378-af5f-732eef3004fd)

## Checklist



  index.md`).
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin




For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
